Project/Managerclass.cs

```diff
@@ -8,14 +8,16 @@ public class Manager : MonoBehaviour
     private sex;
     private string level;
     private int cost;
+    private float initialIncome;
     
 
-    public Manager(string setLevel, int setCost, string setName, string setSex)
+    public Manager(string setLevel, int setCost, string setName, string setSex, float setIncome)

```

Comment:
Change level to int, add constructor that doesn't take in Name/Sex

Reply:
Or just update the default constructor (`Manager()`) to set the costs/income to a default value